app/models/aleph_item.rb

```diff
@@ -54,7 +54,14 @@ def process_item(item, oclc, scan)
       available?: available?(item),
       label: label(item),
       description: description(item),
-      buttons: ButtonMaker.new(item, oclc, scan).all_buttons }
+      buttons: all_buttons(item, oclc, scan) }
+  end
+
+  def all_buttons(item, oclc, scan)
```

Comment:
There should be some comment reminding future-us that if we add any types of button, we MUST LIST THEM HERE or else it's not-🌈 .

Reply:
👍